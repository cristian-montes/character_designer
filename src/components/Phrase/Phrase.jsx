
export default function Phrase({phrases}){
    return(
        <div>
            {phrases.map((phrase) => (
                <p style={{ color:'white'}}>{phrase}</p>
            ))}
        </div>
    )
}