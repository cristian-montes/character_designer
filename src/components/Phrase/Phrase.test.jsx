import { render} from "@testing-library/react";
import Phrase from './Phrase';

describe('Shows Phrases', ()=> {
    it('shuold render Phrases Component', ()=> {
        const {container} = render(
            <Phrase phrases={['Hello My Brudda', 'My friend', 'Potatoes']} />
        )
    
        expect(container).toMatchSnapshot();
    });

})