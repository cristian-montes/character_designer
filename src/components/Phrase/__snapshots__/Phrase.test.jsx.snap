// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shows Phrases shuold render Phrases Component 1`] = `
<div>
  <div>
    <p>
       The head has been changed 
    </p>
    <p>
       The body has been changed 
    </p>
    <p>
       The bottom has been changed 
    </p>
    <p
      style="color: white;"
    >
      Hello My Brudda
    </p>
    <p
      style="color: white;"
    >
      My friend
    </p>
    <p
      style="color: white;"
    >
      Potatoes
    </p>
  </div>
</div>
`;
